docs/source/en/advanced/loader.md

```diff
@@ -1,11 +1,11 @@
 title: Loader
 ---
 
-The most important of Egg enhance Koa is Egg based on a certain agreement, code will be placed in different directories according to the functional differences, it significantly reduce development costs. Loader supports this set of conventions and abstracts that many low-level APIs could be extended.
+The most importance of Egg's enhancements Koa is that Egg is based on a certain agreement, code will be placed in different directories according to the functional differences, it significantly reduces development costs. Loader supports this set of conventions and abstracts that many low-level APIs could be extended.
 
 ## Application, Framework and Plugin
 
-Egg is a low-level framework, applications could use it directly, but Egg only has a little default plugins, applications need to configure plugins to extend features, such as MySQL.
+Egg is a low-level framework, applications could use it directly, but Egg only has a few default plugins, we need to configure plugins to extend features, such as MySQL.
```

Comment:
1) plugin has its own plural form, so this should be `few` instead of `little`.
2) In Chinese we can say `应用需要自己配置插件增加各种特性`, however this is sick in English, we should use `we` as the subject instead of `application` itself. This means `we can extend plugins……in application`.

Reply:
但是你没有加上 in application，导致语义偏离原来的中文意思了